xdsl/backend/riscv/register_allocation.py

```diff
@@ -209,23 +186,21 @@ class RegisterAllocatorJRegs(RegisterAllocator):
 
     def __init__(self, limit_registers: int = 0) -> None:
         self.idx = 0
+        self._register_types = (RegisterType, FloatRegisterType)
         _ = limit_registers
 
     def allocate_registers(self, module: ModuleOp) -> None:
         """
         Sets unallocated registers to an infinite set of `j` registers
         """
         for op in module.walk():
+            # do not allocate registers on non-RISCV-ops
```

Comment:
```suggestion
            # Do not allocate registers on non-RISCV-ops
```

Reply:
I think we mostly try to have the first letter of comments be capital, with a full stop in doc strings and no full stop in regular comments.